Themes/catppuccin-frappe.theme.json

```
@@ -0,0 +1,171 @@
+{
+  "name": "Catppuccin Frappé",
+  "style": "Dark",
+  "author": [
+    {
+      "name": "Roger Steve Ruiz",
+      "url": "https://rog.gr/",
+      "basedOn": "https://github.com/catppuccin/catppuccin"
```

I think it'd be nice to link to the repository itself.

```suggestion
      "basedOn": "https://github.com/catppuccin/the-archive"
```